docs/eog/eog-in-practice/cw21/event-project-stage/eps-sponsorship.rst

```diff
@@ -3,16 +3,16 @@
 CW21 Sponsorship
 =================
 
-We adapted the `sponsorship packages from CW20 <https://software.ac.uk/cw20/sponsorship>`_ to add more value to sponsors in an virtual event and summarised the packages on the `CW21 website <https://software.ac.uk/cw21/sponsorship>`_.
+We adapted the `sponsorship packages from CW20 <https://software.ac.uk/cw20/sponsorship>`_ to add more value to sponsors in a virtual event and summarised the packages on the `CW21 website <https://software.ac.uk/cw21/sponsorship>`_.
 We kept the tiered packages of Silver, Gold and Platinum, with increasing benefits respectively.
 Benefits we added include:
 
-- Place on CW21 Steering Committee
-- Increased number of free registrations
-- Slide shown during breaks
-- Dedicated breakout room for people to speak to you during the breaks at the event
-- Lightning talk slot
-- Video interview with Sponsor shared via social media (Platinum)
+- Place on CW21 Steering Committee.
+- Increased number of free registrations.
+- Slide shown during breaks.
+- Dedicated breakout room for people to speak to you during the breaks at the event.
+- Lightning talk slot.
+- Video interview with Sponsor shared via social media (Platinum).
 
```

Comment:
We don't use full stops at the end of bulleted lists for sentence fragments - see - https://github.com/softwaresaved/event-organisation-guide/blob/master/STYLE.md